src/Application.php

```diff
@@ -132,21 +133,13 @@ public function getConfig()
      * router. Attaches the ViewManager as a listener. Triggers the bootstrap
      * event.
      *
-     * @param array $listeners List of listeners to attach.
      * @return Application
      */
-    public function bootstrap(array $listeners = [])
+    public function bootstrap()
     {
         $serviceManager = $this->serviceManager;
         $events         = $this->events;
 
-        // Setup default listeners
-        $listeners = array_unique(array_merge($this->defaultListeners, $listeners));
-
-        foreach ($listeners as $listener) {
-            $serviceManager->get($listener)->attach($events);
-        }
```

Comment:
Why are you removing the ability to pass additional listeners to `bootstrap()`?

To be clear, I'm fine with it (I don't think we document it anywhere), but you'll need to detail it as a feature removal in the changelog as well.

Reply:
Extra listeners are moved to the application factory since `Application::init()` is going to be dropped.

real change here is that extra listeners to register would migrate from the `application.config.php` to the merged config